src/components/Advertisements/AdvertisementsMocks.ts

```
@@ -11,16 +11,14 @@ import {
-import { ADD_ADVERTISEMENT_MUTATION } from '../../GraphQl/Mutations/mutations';
+} from 'GraphQl/Queries/Queries';
+import { ADD_ADVERTISEMENT_MUTATION } from 'GraphQl/Mutations/mutations';
 
 const { getItem } = useLocalStorage();
 
 export const httpLink = new HttpLink({
   uri: BACKEND_URL,
-  headers: {
-    authorization: 'Bearer ' + getItem('token') || '',
-  },
+  headers: { authorization: 'Bearer ' + getItem('token') || '' },
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)